command/job_run.go

```diff
@@ -317,6 +317,7 @@ func (c *JobRunCommand) Run(args []string) int {
 		PolicyOverride: override,
 		PreserveCounts: preserveCounts,
 		EvalPriority:   evalPriority,
+		Submission:     sub,
```

Comment:
Could we add a `-drop-source` flag (or similar) to allow people to opt out of submitting the source? I'm thinking it might be useful for batch/sysbatch jobs where you might be submitting a relatively large number in an automated way and don't want the overhead of the source.

Reply:
From offline discussion it's become apparent we need to add something to jobspecs to say "Don't save me!" so that operators don't have to change _how_ they submit jobs depending on whether or not they want the source persisted.